### PR TITLE
Add well known lengths to dictionary

### PR DESCRIPTION
**Pull request by andmiguedev1**

### CODE REVIEW

File: server-api/data/length.ts [hunk 1]
```diff
@@ -2,8 +2,56 @@ import { Units } from "../models/converter.model";
...
+  name: "feet",
+  type: "length",
+  factor: 3.2808,
+};
+units["in"] = {
+  unit: "in",
+  base: "m",
+  name: "inch",
+  type: "length",
+  factor: 39.7,
```
Comment (DonaldMurillo) [inline server-api/data/length.ts]: @andmiguedev  one inch is 39.7 meters?????

Comment (andmiguedev1) [reply server-api/data/length.ts]: Yes that doesn't make sense, let me update factor value for Inches

File: server-api/data/length.ts [hunk 2]
```diff
@@ -2,8 +2,56 @@ import { Units } from "../models/converter.model";
 
 const units: Units = {};
 
-units['m'] = {unit:'m', base: 'm', name: 'meter', type: 'length', factor: 1};
-units['cm'] = {unit:'cm', base: 'm', name: 'centimeter', type: 'length', factor: 0.01};
-units['mm'] = {unit:'mm', base: 'm', name: 'milimeter', type: 'length', factor: 0.001};
+units["yard"] = {
```
Comment (DonaldMurillo) [inline server-api/data/length.ts]: @andmiguedev  make the name of the property (units['yd']) the unit not the name

Comment (andmiguedev1) [reply server-api/data/length.ts]: Check last changes under Done, I have updated key to be the abbreviation and accurate factor for current lengths

File: .gitignore [hunk 3]
```diff
@@ -1 +0,0 @@
-node_modules/
```
Comment (DonaldMurillo) [inline .gitignore:1]: @andmiguedev why was this deleted?

Comment (andmiguedev1) [reply .gitignore:1]: I run npm install in a folder that didn't have package.json by mistake. If you see it is not a valid .gitignore for React

Comment (andmiguedev1) [reply .gitignore:1]: If you want we can review this next week

**Review by DonaldMurillo — APPROVED** [review]
